datasource/virtualmachine/data.go

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type Config,Tag,DatasourceOutput
+package virtualmachine
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2/hcldec"
+	"github.com/hashicorp/packer-plugin-sdk/common"
+	"github.com/hashicorp/packer-plugin-sdk/hcl2helper"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer-plugin-sdk/template/config"
+	vsphere "github.com/hashicorp/packer-plugin-vsphere/builder/vsphere/common"
+	"github.com/hashicorp/packer-plugin-vsphere/datasource/common/driver"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// HCL Example:
+//
+// ```hcl
+//
+//	tag {
+//	  category = "team"
+//	  name = "operations"
+//	}
+//	tag {
+//	  category = "sla"
+//	  name = "gold"
+//	}
+//
+// ```
```

Comment:
```suggestion
```

Reply:
Do you mean remove the example from the description to leave only one copy in the example?

Reply:
You can disregard.